src/modules/fundingManager/oracle/LM_ManualExternalPriceSetter_v1.sol

```diff
@@ -0,0 +1,223 @@
+ *                token decimals
+ *              - When getting prices: Output values will be in issuance 
+ *                token decimals
+ *
+ * @custom:security-contact security@inverter.network
+ *                          In case of any concerns or findings, please refer
+ *                          to our Security Policy at security.inverter.network
+ *                          or email us directly!
+ *
+ * @custom:version   v1.0.0
+ *
+ * @custom:standard-version  v1.0.0
+ *
+ * @author  Zealynx Security
+ */
+contract LM_ManualExternalPriceSetter_v1 is
+    ILM_ManualExternalPriceSetter_v1,
+    Module_v1
+{
+    // -------------------------------------------------------------------------
+    // Constants
+
+    /// @notice Role identifier for accounts authorized to set prices
+    /// @dev    This role should be granted to trusted price feeders only
+    bytes32 public constant PRICE_SETTER_ROLE = "PRICE_SETTER_ROLE";
+
+    /// @notice Number of decimal places used for internal price 
+    ///         representation
+    /// @dev    All prices are normalized to this precision for consistent
+    ///         calculations regardless of input/output token decimals.
+    uint8 private constant INTERNAL_DECIMALS = 18;
+
+    // -------------------------------------------------------------------------
+    // State Variables
+
+    /// @notice The price for issuing tokens (normalized to
+    ///         INTERNAL_DECIMALS).
+    uint private _issuancePrice;
+
+    /// @notice The price for redeeming tokens (normalized to
+    ///         INTERNAL_DECIMALS).
+    uint private _redemptionPrice;
+
+    /// @notice Decimals of the collateral token (e.g., USDC with 6 decimals).
+    /// @dev    This is the token used to pay/buy with.
+    uint8 private _collateralTokenDecimals;
+
+    /// @notice Decimals of the issuance token (e.g., ISS with 18 decimals).
+    /// @dev    This is the token being bought/sold.
+    uint8 private _issuanceTokenDecimals;
+
+    /// @dev    Storage gap for upgradeable contracts.
+    uint[50] private __gap;
+
+    // -------------------------------------------------------------------------
+    // Initialization
+
+    /// @inheritdoc Module_v1
+    function init(
+        IOrchestrator_v1 orchestrator_,
+        Metadata memory metadata_,
+        bytes memory configData_
+    ) external override(Module_v1) initializer {
+        __Module_init(orchestrator_, metadata_);
+
+        // Decode collateral and issuance token addresses from configData_.
+        (address collateralToken, address issuanceToken) =
+            abi.decode(configData_, (address, address));
+
+        // Store token decimals for price normalization
+        _collateralTokenDecimals = IERC20Metadata(collateralToken).decimals();
+        _issuanceTokenDecimals = IERC20Metadata(issuanceToken).decimals();
+    }
+
+    //--------------------------------------------------------------------------
```

Comment:
Missing space here as well in this contract. Line length may need an adjustment afterwards.

Reply:
fixed